deploy/template.yaml

```diff
@@ -325,6 +325,18 @@ Resources:
               Value: !GetAtt ApiGwHttpEndpoint.ApiEndpoint
             - Name: REDIS_SESSION_URL
               Value: !GetAtt RedisCache.RedisEndpoint.Address
+            - Name: GTM_ID
+            Value:
```

Comment:
The formatting isn't quite right here. Value needs to indent to align with `Name` and the function placement is slightly off, this should work. You're also missing a `!Sub` on line 339 to sub in the `${Environment}` param
```
            - Name: GTM_ID
              Value: !If 
                - IsProduction
                - GTM-TT5HDKV
                - GTM-TK92W68
            - Name: ANALYTICS_DOMAIN
              Value: !If
                - 'IsProduction'
                - 'account.gov.uk'
                - !Sub '${Environment}.account.gov.uk'
```

I'd highly recommend installing `cfn-lint` in your IDE of choice.

Reply:
@danworth Ooh, thanks!

re `cfn-lint` - that's a good call. However, we can't ensure that every IDE has that setup, but we can make sure it is run on every PR via GHAs. However, it looks like the installation isn't going to be clean, so we might need a `dev` image to run our code against.

Reply:
Yeah Vam added `cfn-lint` to a `pre-commit` hook and we're running checkov and lint on our PRs via a GHA. `cfn-lint` in VSCode catches most of my errors before I get so far as committing.